plugins/joke.lua

```diff
@@ -13,8 +13,8 @@ return {
 			if not first then
 				first = sender[1]
 			end
-			local h, s = assert(http_request.new_from_uri(
-			        "https://api.icndb.com/jokes/random?limitTo=nerdy&firstName="..first.."&lastName="):go())
+			local h, s = http_request.new_from_uri(
```

Comment:
To refactor a call to `assert()` in Lua to make it "safe" (so that it doesn't error), it is best to just check if the value is falsy (`if not h then`) and if so, `return false, "error string here"`.